src/apps/common/submodules/dragableUploads/dragableUploads.js

```diff
@@ -0,0 +1,216 @@
+define(function(require) {
+	var $ = require('jquery'),
+		_ = require('lodash'),
+		monster = require('monster');
+
+	var dragableUploads = {
+		// Defines API requests not included in the SDK
+		requests: {},
+
+		// Define the events available for other apps
+		subscribe: {
+			'common.dragableUploads.renderUploadArea': 'renderUploadArea'
+		},
+
+		appFlags: {
+			dragableUploads: {
+				files: [],
+				maxFileSize: 5242880 // 5MB
+			}
+		},
+
+		renderUploadArea: function(args) {
```

Comment:
Don't forget to append the submodule's name to each of its methods.

Reply:
oops, thanks!